department-of-veterans-affairs/va.gov-cms
# VAGOV-5582 Makes Behat DST work again

## What this does
- Moves `/tests/drupal-spec-tool` to `/tests/behat/drupal-spec-tool` - DST are behat tests, so this is where they should live.
- Autoloads CustomDrupal namespace necessary for custom behat steps
- Updates behat.yml to function properly, autoload DST classes, and look inside `/drupal-spec-tool` directory

## QA
 - cd `/tests`
 - `lando behat --tags=@access` (this runs a DST test)
 - Confirm test runs (see screenshot)
 - `lando behat --tags=@errors` (this runs a behat test in `/features` - note that a step is missing, but this is expected)
 - Confirm test runs (see screenshot)

## Screenshots
![Screenshot_2019-08-17_11-39-34](https://user-images.githubusercontent.com/2404547/63214274-31144800-c0e4-11e9-9c68-da6714df0f83.png)
![Screenshot_2019-08-17_11-43-01](https://user-images.githubusercontent.com/2404547/63214275-31144800-c0e4-11e9-8b06-9337cff5e865.png)
